tests/queries/0_stateless/02710_allow_suspicious_indices.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS allow_suspicious_indices;
+CREATE TABLE allow_suspicious_indices (id UInt32) ENGINE = MergeTree() order by (id, id) settings allow_suspicious_indices = 1;
+DROP TABLE IF EXISTS allow_suspicious_indices;
+CREATE TABLE allow_suspicious_indices (id UInt32) ENGINE = MergeTree() order by (id, id) settings allow_suspicious_indices = 0;  -- { serverError BAD_ARGUMENTS }
+DROP TABLE IF EXISTS allow_suspicious_indices;
+CREATE TABLE allow_suspicious_indices (id UInt32, index idx (id, id) TYPE minmax GRANULARITY 1) ENGINE = MergeTree() order by id settings allow_suspicious_indices = 1;
```

Comment:
`GRANULARITY <n>` is optional these days

Reply:
Added

Reply:
I meant you can remove `GRANULARITY`, so l. 6 becomes:
```sql
CREATE TABLE allow_suspicious_indices (id UInt32, index idx (id, id) TYPE minmax) ENGINE = MergeTree() order by id settings allow_suspicious_indices = 1;
```